docs/OnTrack/Deployment/Google Cloud/Index.md

```diff
@@ -0,0 +1,35 @@
+# Google Cloud - Documentation Index
+
+## [Overview](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/Overview.md)
+
+Google Cloud team's overview and high-level documentation.
+
+## [Research & Findings](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/Research%20%26%20Findings/google-cloud-research.md)
+
+Google Cloud team's research and findings -
+
+- Aims
+- Findings
+- Learnings
+- Outcomes
+- References
+- Links
+
+## Links
+
+- [GitHub repo for Thoth-Tech/Doubtfire-deploy](https://github.com/thoth-tech/doubtfire-deploy)
+- [GitHub repo for Thoth-Tech/Doubtfire-deploy/DEPLOYING.md](https://github.com/thoth-tech/doubtfire-deploy/blob/main/DEPLOYING.md)
+- [Trello board for Deployment - Google Cloud epic](https://trello.com/b/dI1yx9A1/deployment)
+- [GitHub repo for Thoth-Tech/Doubtfir-deploy-GCP](https://github.com/thoth-tech/doubtfire-deploy-GCP)
+- [Miro for Google Cloud - Doubtfire](https://miro.com/app/board/uXjVO0h8ZSE=/?share_link_id=62396987373)
+- [Miro for Google Cloud - CI/CD](https://miro.com/app/board/uXjVO64xoQw=/?share_link_id=57734801709)
+
+## Assets
+
+- [Google Cloud - Doubtfire](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/GoogleCloud_Doubtfire.jpg)
+- [Google Cloud - CI/CD](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/GoogleCloud_CICD.jpg)
+- [doubtfire-localhost-compose](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/doubtfire-localhost-compose.png)
+- [doubtfire-localhost-docker](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/doubtfire-localhost-docker.png)
+- [doubtfire-localhost](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/doubtfire-localhost.png)
+- [gcp-instance-docker](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/gcp-instance-docker.png "Instance running docker containers")
+- [ontrack-login](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/ontrack-login.png "OnTrack Login")
```

Comment:
**suggestion:** I think we only need to link to the main content files, aka. `.md`. These assets only make sense in the context of those markdown content. Additionally, it also increases maintenance effort. I suggest dropping this section. What do you think?

Reply:
Sounds good. I've dropped it.